dapps/react-dapp-v2/src/constants/default.ts

```diff
@@ -133,6 +135,16 @@ type RelayerType = {
   label: string;
 };
 
+/**
+ * KADENA
+ */
+export enum DEFAULT_KADENA_METHODS {
```

Comment:
It'd be nice if there was some standard to define the methods available. Here is the closest thing to a standard: <https://github.com/kadena-io/KIPs/blob/jam/wallet-connect/kip-0017.md#default-methods>, and there they say:

* kadena_sign_v1
* kadena_quicksign_v1

A dapp I know of supports these two: 'kadena_sign', 'kadena_quicksign', although quick sign isn't actually used at all.

What is the standard?

Reply:
I'm talking about these:
 KADENA_SIGN_TRANSACTION = "kadena_signTransaction",
   KADENA_SIGN_MESSAGE = "kadena_signMessage",